### DISCUSSION TITLE
Docs for database migrations

### DISCUSSION THREAD

**Discussion by vimkaf** [leafsphp/leaf#179] [General]
I can't for the life of me find the docs for database migrations. It would be really helpful if anyone can provide a link to the database migration docs.

**Comment by iammiloslukic** [comment]
I haven't find the docs myself, but there are some example files in the project already so you can check those.
Anyhow, I usually check Laravel's docs for reference.
Hope it helps. :)

**Reply by vimkaf** [reply]
Yeah I later found it [here](https://api.leafphp.dev/docs/database/migrations.html)

**Reply by mychidarko** [reply]
Thanks for pointing this out @vimkaf, the documentation is currently being rewritten. I'll be sure to update this
